### PR TITLE
pot/drip mapping + test

### PR DESCRIPTION
**Pull request by ptPierre**

### CODE REVIEW

File: tests/mappings/modules/rates/pot/handleDrip.test.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+describe('Pot#handleDrip', () => {
+  test('Updates SystemState.rateAccumulator', () => {
+    let event = createEvent()
+    mockChi()
+    mockDebt()
+
+    handleDrip(event)
+
+    assert.fieldEquals('SystemState', 'current', 'rateAccumulator', '10')
+
```
Comment (llopez) [inline tests/mappings/modules/rates/pot/handleDrip.test.ts:28]: please check `lastDripCall` is updated

File: src/mappings/modules/rates/pot.ts [hunk 2]
```diff
@@ -66,3 +66,16 @@ export function handleExit(event: LogNote): void {
...
+
+export function handleDrip(event: LogNote): void {
+  let system = systemModule.getSystemState(event)
+  let potContract = Pot.bind(Address.fromString('0x197e90f9fad81970ba7976f33cbd77088e5d7cf7'))
+
+  let callResult = potContract.try_chi()
+  if (callResult.reverted) {
+    log.warning('handleDrip: try_chi reverted', [])
+  }
+  system.rateAccumulator = callResult.value
```
Comment (llopez) [inline src/mappings/modules/rates/pot.ts:78]: I guess we need to update the value of `ratesAccumulator` with the result of `tmp = rmul(rpow(dsr, now - rho, ONE), chi);`

File: subgraph.yaml [hunk 3]
```diff
@@ -516,6 +516,11 @@ dataSources:
           topic0: '0x7f8661a100000000000000000000000000000000000000000000000000000000'
           handler: handleExit
 
+        # function drip(uint tab)
```
Comment (llopez) [inline subgraph.yaml:503]: no parameters here

**Review by llopez — CHANGES_REQUESTED** [review]

File: schema.graphql [hunk 4]
```diff
@@ -390,6 +390,12 @@ type SystemState @entity {
   " Total savings in DAI of all users (Pot#Pie) "
   totalSavingsInPot: BigDecimal!
 
+  " Value which decides how much dai - given when drip() is called "
+  rateAccumulator: BigInt
+
+  " last time the drip() function was called "
+  lastDripCall: BigInt
+
```
Comment (rtomas) [inline schema.graphql]: I suggest adding the name of the contract to the description and to the var name. There is another drip in the Jug contract.

**Review by llopez — APPROVED** [review]

**Pull request merged**